browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -54,17 +54,16 @@ RegisterStyleOverride(
       :host {
         --brave-settings-menu-margin-v: 24px;
         --brave-settings-menu-padding: 24px;
-        --settings-nav-item-color: #424242 !important;
+        --settings-nav-item-color: var(--leo-color-text-primary) !important;
         position: sticky;
         top: var(--brave-settings-menu-margin-v);
-        margin: 0 var(--brave-settings-menu-margin) !important;
+        margin: 0 !important;
         max-height: calc(100vh - 56px - (var(--brave-settings-menu-margin-v) * 2) - (var(--brave-settings-menu-padding) * 2));
         min-width: 172px;
+        max-width:250px;
         border-radius: 6px;
-        background-color: #fff;
         overflow-y: auto;
         padding: 24px !important;
-        box-shadow: var(--leo-effect-elevation-01);
```

Comment:
we don't need the shadow anymore?

Reply:
No because it's now docked to the side, no need for shadow